test/DynamoCoreWpfTests/WorkspaceSaving.cs

```diff
@@ -791,6 +791,37 @@ public void CustomNodeSaveAsAddsNewLoadedNodeToCustomNodeManager()
             Assert.IsTrue(ViewModel.Model.CustomNodeManager.Contains(oldId));
         }
 
+        [Test]
+        public void CustomNodeEditNodeDescriptionKeepingViewBlockInDyf()
+        {
+            // new custom node
+            // Set file path
+            // custom node update description and node name
+            // Check if serialized workspace still have view block
+
+            var dynamoModel = ViewModel.Model;
+            var nodeName = "Cool node";
+            var catName = "Custom Nodes";
+            var initialId = Guid.NewGuid();
+
+            var def = dynamoModel.CustomNodeManager.CreateCustomNode(nodeName, catName, "", initialId);
+            var workspace = (CustomNodeWorkspaceModel)def;
+            workspace.FileName = GetNewFileNameOnTempPath("dyf");
+
+            Dynamo.Wpf.FunctionNodeViewCustomization temp = new Dynamo.Wpf.FunctionNodeViewCustomization();
+            FunctionNamePromptEventArgs args = new FunctionNamePromptEventArgs();
+            args.Success = true;
+            args.Name = "Cool node_v2";
+            args.Category = "Custom Nodes";
+            args.Description = "test";
+
+            temp.EditCustomNodeProperties(null, args);
```

Comment:
@ramramps @mjkkirschner I was able to locate some sample unit tests using left mouse click but I failed to find one using right click.. Do you guys know if there is good way of testing handler somehow? I feel like I am not doing this right..